xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/objects/classes/TextAreaClass.java

```diff
@@ -381,13 +381,19 @@ public void displayView(StringBuffer buffer, String name, String prefix, BaseCol
                 buffer.append(result);
             }
         } else {
-            StringBuffer result = new StringBuffer();
-            super.displayView(result, name, prefix, object, context);
+            BaseProperty property = (BaseProperty) object.safeget(name);
+            String content;
+            if (property != null) {
+                content = property.toText();
```

Comment:
Similar comment than above but with the following `if` instead of just a buffer.append (i.e. avoid useless work when the property is null).

Reply:
Else branch removed here too https://github.com/xwiki/xwiki-platform/pull/1315/files#diff-912dd4275a5ac5f2116abef20711b489R384